interface/ceed-qfunctioncontext.c

```diff
@@ -343,6 +343,58 @@ int CeedQFunctionContextSetInt32(CeedQFunctionContext ctx,
   return CEED_ERROR_SUCCESS;
 }
 
+/**
+  @brief Get additional destroy routine for CeedQFunctionContext user data
+
+  @param[in] ctx          CeedQFunctionContext to get user destroy function
+  @param[out] f_mem_type  Memory type to use when passing data into `f`
+  @param[out] f           Additional routine to use to destroy user data
+
+  @return An error code: 0 - success, otherwise - failure
+
+  @ref Backend
+**/
+int CeedQFunctionContextGetDataDestroy(CeedQFunctionContext ctx,
+                                       CeedMemType *f_mem_type, CeedQFunctionContextDataDestroyUser *f) {
+  if (f_mem_type) *f_mem_type = ctx->data_destroy_mem_type;
+  if (f) *f = ctx->data_destroy_function;
+  return CEED_ERROR_SUCCESS;
+}
+
+/**
+  @brief Destroy user data held by CeedQFunctionContext, using function set by
+     CeedQFunctionContextSetDataDestroy, if applicable
+
+  @param[in,out] ctx  CeedQFunctionContext to destroy user data
+
+  @return An error code: 0 - success, otherwise - failure
+
+  @ref Backend
+**/
+int CeedQFunctionContextDestroyData(CeedQFunctionContext ctx) {
```

Comment:
Looks like this function isn't actually used by backends (and might be harmful if it were). Can we make it `static` here and remove from `ceed/backend.h`?

Reply:
Good point. I'll change it on Tuesday